test/steps/v2_steps.py

```diff
@@ -36,17 +26,24 @@
 from algosdk.abi.contract import NetworkInfo
 from algosdk.error import ABITypeError, AlgodHTTPError, IndexerHTTPError
 from algosdk.future import transaction
+from algosdk.testing.dryrun import DryrunTestCaseMixin
 from algosdk.v2client import *
 from algosdk.v2client.models import (
-    DryrunRequest,
-    DryrunSource,
     Account,
     ApplicationLocalState,
+    DryrunRequest,
+    DryrunSource,
 )
-from algosdk.testing.dryrun import DryrunTestCaseMixin
+from behave import (
+    given,
+    register_type,  # pylint: disable=no-name-in-module
+    step,
+    then,
+    when,
+)
+from glom import glom
 
-from test.steps.steps import token as daemon_token
-from test.steps.steps import algod_port
+# TODO: This file is WAY TOO BIG. Break it up into logically related chunks.
```

Comment:
I can take this on as a follow up if you have the time to review it!

Reply:
🙌 sounds good!

Reply:
Here's the PR. I'll publicize it to other interested parties in the morning.

Reply:
Awesome work, thanks for doing this